src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -50,10 +50,12 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
 
   renderFAQAndSpecialist = () => {
     const {
-      artwork: { isAcquireable, isInAuction, sale },
+      artwork: { isAcquireable, isInAuction, sale, availability },
     } = this.props
 
-    if (isInAuction && sale && !sale.isClosed) {
+    const hasNotSold = availability && availability !== "sold"
```

Comment:
Here (and elsewhere in these changes), I would use the metaphysics property [`isForSale` ](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/artwork/index.ts#L360).

So line 58 could just be `if (isInAuction && sale && !sale.isClosed && isForSale)` and you can avoid this extra logic.

`availability` isn't binary, but I think that ☝️ is more correct anyways.